### DISCUSSION TITLE
check possibilities for defining contracts technology/framework independent

### DISCUSSION THREAD

**Discussion by exbachi** [artiso-solutions/adsd-hotel#59] [General]
At the moment we define exposed data structures in a shared contracts project. The problem here is, that it can only be used in .net. We should take a look at possibilities to define these in more open and independent way

**Comment by dpetscher** [comment]
@tommasobertoni mentioned [JSON Schema](https://json-schema.org/). As he said there is proprietary support in [Json.NET Schema](https://www.newtonsoft.com/jsonschema) but there also seems to be an [open source implementation](https://github.com/RicoSuter/NJsonSchema), so maybe that's something worth considering.

**Comment by tziemek** [comment]
Maybe we can use OpenApi for that. This topic would be excellent for a discussion and not an issue. I move it later.
